kuma/javascript/ssr-server.js

```diff
@@ -84,6 +86,15 @@ app.post('/ssr/:componentName', (req, res) => {
     res.json(ssr(req.params.componentName, req.body));
 });
 
+// Important that this is defined *after* the request handlers have been
+// added otherwise Sentry won't automatically hook in.
+if (process.env.SENTRY_DSN) {
+    // The request handler must be the first middleware on the app
+    app.use(Sentry.Handlers.requestHandler());
+    // The error handler must be before any other error middleware
```

Comment:
I'm thinking we should copy the comment from https://docs.sentry.io/platforms/node/express/, so this would become:
```
// The error handler must be before any other error middleware and after all controllers
```